CHANGE-NOTES.html

```diff
@@ -1,11 +1,17 @@
+<h2>Features:</h2>
```

Comment:
`<h2>` apparently makes no difference from regular text (at least in `Plugins` dialog):
![image](https://user-images.githubusercontent.com/3383210/89198702-dce0cd00-d5ad-11ea-96b1-4033185b086e.png)

Reply:
ok, so remove it